arm-web/2016-08-01/swagger/WebApps.json

```diff
@@ -732,6 +732,49 @@
         "x-ms-long-running-operation": true
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/config": {
+      "get": {
+        "tags": [
+          "WebApps"
+        ],
+        "summary": "List the configurations of an app",
+        "description": "List the configurations of an app",
+        "operationId": "WebApps_ListConfigurations",
```

Comment:
Please provide `x-ms-examples` section according to https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/x-ms-examples.md.